src/schema/objects/extensions.yaml

```diff
@@ -0,0 +1,225 @@
+.edf:
+  name: European Data Format
+  description: |
+    A [European data format](https://www.edfplus.info/) file.
+
+    Each recording consists of a single `.edf`` file.
+    [`edf+`](https://www.edfplus.info/specs/edfplus.html) files are permitted.
+    The capital `.EDF` extension MUST NOT be used.
+.eeg:
+  name: BrainVision Binary Data
+  description: |
+    A binary data file in the
+    [BrainVision Core Data Format](https://www.brainproducts.com/productdetails.php?id=21&tab=5).
+    These files come in three-file sets, including a `.vhdr`, a `.vmrk`, and a `.eeg` file.
+.fdt:
+  name: EEGLAB FDT
+  description: |
+    A [EEGLAB](https://sccn.ucsd.edu/eeglab) file.
+
+    The format used by the MATLAB toolbox [EEGLAB](https://sccn.ucsd.edu/eeglab).
+    Each recording consists of a `.set` file with an optional `.fdt` file.
+.fif:
+  name: Functional Imaging File Format
+  description: |
+    An MEG file format used by Neuromag, Elekta, and MEGIN.
+.jpg:
+  name: Joint Photographic Experts Group Format
+  description: |
+    A JPEG image file.
+.json:
+  name: JavaScript Object Notation
+  description: |
+    A JSON file.
+.kdf:
+  name: KRISS KDF
+  description: |
+    A KRISS (file with extension `.kdf`) file.
+
+    Each experimental run on the KRISS system produces a file with extension `.kdf`.
+    Additional files that may be available in the same folder include
+    the digitized positions of the head points (`\_digitizer.txt`),
+    the position of the center of the MEG coils (`.chn`),
+    and the event markers (`.trg`).
+.md:
+  name: Markdown
+  description: |
+    A Markdown file.
+.mefd/:
+  name: Multiscale Electrophysiology File Format Version 3.0
+  description: |
+    A folder in the [MEF3](https://osf.io/e3sf9/) format.
+
+    Each recording consists of a `.mefd` directory.
+.mrk:
+  name: MRK
+  description: |
+    A file containing MEG sensor coil positions.
+
+    Used by KIT, Yokogawa, and Ricoh MEG systems.
+.nii:
+  name: Nifti
+  description: |
+    A nifti file.
+.nii.gz:
+  name: Compressed Nifti
+  description: |
+    A compressed nifti file.
+.nwb:
+  name: Neurodata Without Borders Format
+  description: |
+    A [Neurodata Without Borders](https://nwb-schema.readthedocs.io) file.
+
+    Each recording consists of a single `.nwb` file.
+.ome.btf:
+  name: Open Microscopy Environment BigTIFF
+  description: |
+    A [BigTIFF](https://www.awaresystems.be/imaging/tiff/bigtiff.html) image file, for very large images.
+.ome.tif:
+  name: Open Microscopy Environment Tag Image File Format
+  description: |
+    An [OME-TIFF](https://docs.openmicroscopy.org/ome-model/6.1.2/ome-tiff/specification.html#) image file.
+.png:
+  name: Portable Network Graphics
+  description: |
+    A [Portable Network Graphics](http://www.libpng.org/pub/png/) file.
+.raw:
+  name: RAW
+  description: |
+    When produced by a KIT / Yokogawa / Ricoh MEG system, this file contains stuff.
```

Comment:
I don't know. MEG extensions are not well documented and are very confusing.

Reply:
@sappelhoff do you know what this file contains?

Reply:
perhaps @jasmainak knows? Or maybe he knows who might know?

Reply:
Maybe this helps: https://www.fieldtriptoolbox.org/getting_started/yokogawa/?

"Trial-based evoked fields" is what it says

Reply:
Thanks! I'll use that.